configure.ac

```diff
@@ -322,7 +322,7 @@ AC_ARG_ENABLE([werror],
     [enable_werror=no])
 
 AC_ARG_ENABLE([external-signer],
-    [AS_HELP_STRING([--enable-external-signer],[compile external signer support (default is yes, requires Boost::Process)])],
+    [AS_HELP_STRING([--enable-external-signer],[compile external signer support (default is auto, requires Boost::Process)])],
```

Comment:
the default is auto, you can also supply yes and no. Should we follow the convention used elsewhere of specifying the possible inputs, so for example, the help output could be:

```
--enable-external-signer[=yes|no|auto]
                          compile external signer support (default is auto,
                          requires Boost::Process)
```